NullPointerException ProgressArcView.stop() on LoginFailed

**Describe the bug and how to reproduce**
ACRA crash report by mail
```
APP_VERSION_CODE=100
APP_VERSION_NAME=1.0.0
ANDROID_VERSION=6.0.1
CUSTOM_DATA=
STACK_TRACE=java.lang.RuntimeException: Unable to start activity ComponentInfo{org.piwigo.android/org.piwigo.ui.login.LoginActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.github.jorgecastilloprz.progressarc.ProgressArcView.stop()' on a null object reference
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2416)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476)
	at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:4077)
	at android.app.ActivityThread.-wrap15(ActivityThread.java)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1350)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:148)
	at android.app.ActivityThread.main(ActivityThread.java:5417)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.github.jorgecastilloprz.progressarc.ProgressArcView.stop()' on a null object reference
	at com.github.jorgecastilloprz.FABProgressCircle.hide(FABProgressCircle.java:166)
	at org.piwigo.ui.login.LoginActivity.loginError(LoginActivity.java:145)
	at org.piwigo.ui.login.LoginActivity.lambda$cvhGC3Mv9TzM8IjVDnu38ao367Q(LoginActivity.java)
	at org.piwigo.ui.login.-$$Lambda$LoginActivity$cvhGC3Mv9TzM8IjVDnu38ao367Q.onChanged(lambda)
	at androidx.lifecycle.LiveData.considerNotify(LiveData.java:131)
	at androidx.lifecycle.LiveData.dispatchingValue(LiveData.java:144)
	at androidx.lifecycle.LiveData$ObserverWrapper.activeStateChanged(LiveData.java:442)
	at androidx.lifecycle.LiveData$LifecycleBoundObserver.onStateChanged(LiveData.java:394)
	at androidx.lifecycle.LifecycleRegistry$ObserverWithState.dispatchEvent(LifecycleRegistry.java:361)
	at androidx.lifecycle.LifecycleRegistry.forwardPass(LifecycleRegistry.java:300)
	at androidx.lifecycle.LifecycleRegistry.sync(LifecycleRegistry.java:339)
	at androidx.lifecycle.LifecycleRegistry.moveToState(LifecycleRegistry.java:145)
	at androidx.lifecycle.LifecycleRegistry.handleLifecycleEvent(LifecycleRegistry.java:131)
	at androidx.lifecycle.ReportFragment.dispatch(ReportFragment.java:123)
	at androidx.lifecycle.ReportFragment.onStart(ReportFragment.java:83)
	at android.app.Fragment.performStart(Fragment.java:2244)
	at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1002)
	at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1148)
	at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1130)
	at android.app.FragmentManagerImpl.dispatchStart(FragmentManager.java:1958)
	at android.app.FragmentController.dispatchStart(FragmentController.java:163)
	at android.app.Activity.performStart(Activity.java:6274)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2379)
	... 10 more

```

--- a/app/src/main/java/org/piwigo/ui/login/LoginActivity.java
+++ b/app/src/main/java/org/piwigo/ui/login/LoginActivity.java
@@ -106,7 +106,8 @@ public class LoginActivity extends BaseActivity {
         viewModel.loadAccount(account);
     }
 
-    public void onClick(View v) {
+    public void onClick(View v)
+    {
         viewModel.onLoginClick(fabProgressCircle);
     }
 
@@ -127,7 +128,8 @@ public class LoginActivity extends BaseActivity {
     }
 
     private void loginSuccess(LoginResponse response) {
-        fabProgressCircle.hide();
+        if (fabProgressCircle != null)
+            fabProgressCircle.hide();
         if (viewModel.isEditExisting()) {
             finish();
         } else if (userManager.userExists(response.url, response.username)) {
@@ -142,7 +144,8 @@ public class LoginActivity extends BaseActivity {
     }
 
     private void loginError(Throwable throwable) {
-        fabProgressCircle.hide();
+        if (fabProgressCircle != null)
+            fabProgressCircle.hide();
         String msg;
         URI uri;
         String host = viewModel.url.get();

